lib/services/post.ts

```diff
@@ -38,13 +43,14 @@ type UpdatablePostResolved = {
 export async function getPost(
   id: string,
   includeUser: boolean
-): Promise<Post | PostWithUser | null> {
+) : Promise<PostWithUserWithBook | null> {
```

Comment:
Is this a change to the API? If so, I think it might be worthwhile updating the open api spec as well.

Reply:
Is the book data already included in the spec and I just missed it when implementing? If so, then I'll submit a bug and I can remedy in a quick PR.

Reply:
Yeah the book data should be included

Reply:
Okay. I'll make a bug and assign myself. Should be a short PR, I'll try to get it done today.

Reply:
Okay. I've merge my change as a separate PR. You should be good to remove this from your PR.